src/components/FilterPanel/FilterPanel.styles.ts

```diff
@@ -0,0 +1,81 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { styled } from '@mui/material/styles';
+
+import { FILTER_BUTTON_SIZES } from './FilterPanel.constants';
+
+import type { FilterVariant } from './FilterPanel.types';
+import type { Theme } from '@mui/material/styles';
+
+interface FilterPanelContainerProps {
+  panelVariant: 'primary' | 'secondary';
+  size?: 'medium' | 'large' | 'xlarge';
+}
+
+export const FilterPanelContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'panelVariant',
```

Comment:
shouldForwardProp callback currently returns prop !== 'panelVariant'. That function will forward all other props to the DOM; ensure any non-DOM props are excluded to prevent React warnings.

**Sugerencia:**
```typescript
If other custom props exist (like size, selected), list them here. Example:

shouldForwardProp: (prop) => !['panelVariant', 'size', 'selected'].includes(prop as string),
```